command/analyzer/verify/verify.go

```diff
@@ -140,12 +141,17 @@ func (a *AnalyzerVerifyOpts) verifyAnalyzer() (err error) {
 	// Check for validation errors in analyzer issues and display them (if any)
 	if issuesValidationErrors != nil && len(*issuesValidationErrors) > 0 {
 		configurationValid = false
-		a.Spinner.StopSpinnerWithError("Failed to validate the following issues", err)
+		a.Spinner.StopSpinnerWithError("Failed to validate the issues", err)
 		for _, validationError := range *issuesValidationErrors {
-			fmt.Printf("  > %s\n", validationError.File)
-			for _, err := range validationError.Errors {
-				failureMsg := utils.GetBulletMessage(err.Message, "red")
-				fmt.Printf("    %s\n", failureMsg)
+			// Get diagnostics.
+			reportedDiagnostics, err := diagnostics.GetDiagnostics(validationError)
+			if err != nil {
+				a.Spinner.StopSpinnerWithError("Failed to validate the following issues", err)
+			}
+
+			// Print diagnostics to the console.
+			for _, diagnostic := range reportedDiagnostics {
+				fmt.Printf("%s\n", diagnostic)
```

Comment:
@burntcarrot Here, we need to also show the issue TOML name as well where the validation failed else the author wouldn't get to know for which issue the validation failed. We can do it something like this:
```
[✗] Failed to validate the following issues
  > <issue-shortcode>.toml
    • Validation message
    <Diagnostic>
    • Next validation message
    <Diagnostic>
  > <another-issue-shortcode>.toml
    • Validation message
    <Diagnostic>  
```
Also, please add tests? Thanks!

Reply:
Updated screencast: https://asciinema.org/a/ZWGEk9clGcd80JvYC0MzOgOja

I've updated the unit tests.

Reply:
Thanks! Can you please add a few tests for issue TOMLs as well?
Adding to that, can we add some bullet before the filename and indent the validation errors under it? Also, please use different color for filenames. Helps in easy identification. Here is how we display it as of now:

<img width="407" alt="Screenshot 2022-07-19 at 4 32 18 PM" src="https://user-images.githubusercontent.com/68370997/179735329-2760ba02-a709-49ef-805e-3e57b1ee71fc.png">

Reply:
Resolved in https://github.com/deepsourcelabs/cli/pull/176/commits/e8afc2df93e4517627797c15919560da73441850 and https://github.com/deepsourcelabs/cli/pull/176/commits/1f0213bb1d54781a76eea90fa5f5160017b54c5c.